lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -568,11 +574,18 @@ namespace Js
 
     ArrayBuffer* ArrayBuffer::FromVar(Var aValue)
     {
-        AssertMsg(Is(aValue), "var must be an ArrayBuffer");
+        AssertOrFailFastMsg(Is(aValue), "var must be an ArrayBuffer");
 
         return static_cast<ArrayBuffer *>(RecyclableObject::FromVar(aValue));
```

Comment:
This can be UnsafeFromVar, since there is already failfast right above

Reply:
Looks like a number of the FromVars have this. But seems like in some they just directly static_cast a var to the type. If that's possible, it seems preferable here

---
In reply to: [147226615](https://github.com/Microsoft/ChakraCore/pull/4040#discussion_r147226615) [](ancestors = 147226615)

Reply:
Makes sense. I'll try to make it uniform across FromVars, either with or without RecyclableObject::UnsafeFromVar